Mondee Watch App/Model/GameResultModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  MessageKey.swift
```

Comment:
별거 아니지만,, 파일이름이 달라서 Suggestion 합니다!
```suggestion
//  GameResultModel.swift
```